test/tsconfig.json

```diff
@@ -142,4 +143,4 @@
         "rule-tester/testData.ts",
         "rule-tester/utilsTests.ts"
     ]
-}
+}
```

Comment:
Actually, ignore this, it seems that grunt always removes this newline, so lets just get it removed and keep it removed